dts/bindings/sensor/sensor-device.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Sensor Device
+
+include: base.yaml
+
+properties:
+  vendor:
```

Comment:
If we want `vendor` and such.  We can expose this from edtlib and gen_defines.py.  We have a lot of the logic already in `_check_compatible` function in edtlib.py.

Reply:
I like this idea, thanks.